src/DiffOpt.jl

```diff
@@ -1,13 +1,84 @@
 module DiffOpt
 
 using Random
+using LinearAlgebra
 using MathOptInterface
 
 const MOI = MathOptInterface;
 const MOIU = MathOptInterface.Utilities;
 
+include("./gen_random_problem.jl")
 include("./utils.jl")
 
+
+function DiffModel(_model::MOI.AbstractOptimizer, con_idx)
```

Comment:
use snake case, like `diff_model` instead of DiffModel, which is kept for types and modules.
Put this function in a different file, and add a docstring for it

Reply:
will do that